src/shared/md/2010-07-22-magic-increasing-number.md

```diff
@@ -6,4 +6,10 @@ Look at me, I'm the magic increasing number!
 ```
 
 
---- [@thomasfuchs](http://twitter.com/thomasfuchs)
\ No newline at end of file
+--- [@thomasfuchs](http://twitter.com/thomasfuchs)
+
+That's because `9999999999999999` > [`Number.MAX_SAFE_INTEGER`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER). That's the highest integer possible without loosing precision. You can check if your integer is save using [`Number.isSafeInteger()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isSafeInteger).
```

Comment:
without **losing** precision
check if your integer is **safe**

Reply:
Oh, shoot. That's embarrassing. Thanks for that hint! I made a new [pull request](158) since this one is already closed.